src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -426,11 +438,14 @@ export default class BrowseSkill extends React.Component {
             this.loadLanguages();
           },
         );
-        // console.log(self.state)
       },
       error: function(e) {
-        console.log('Error while fetching skills', e);
-        return self.loadCards();
+        // console.log('Error while fetching skills', e);
+        // return self.loadCards();
+        self.setState({
+          skillsLoaded: true,
+        });
+        self.openSnackbar('You are offline!');
```

Comment:
@Akshat-Jain @akshatnitd Can we have a discussion on this. Coz the alternative that we had was that it would keep making ajax requests, which is definitely not desirable as it ends up slowing down the browser after a while. This is coming from a guy using 8 gigs of RAM btw.